rdm/checklists/62304_2006-05_class_a.txt

```diff
@@ -0,0 +1,6 @@
+# Audit checklist for IEC62304 version 2006-05 Class A products
```

Comment:
What is 2006-05 vs 2006_AMD?

Reply:
Two versions of the standard. Not a lot of changes between them, but some additions and subtractions. Plus a few places with significant rewordings.

Reply:
So is 2006-05 different from just 2006?

Reply:
Ahh, maybe I am confused and there is no "2006" but only "2006-05" and "2006_AMD". Where does the "-05" come from?

Reply:
The phrase "2006-05" appears on the title page. But we should use whatever names will make the most sense to people. Suggestions?

Reply:
I suggest "2006" and "2006+AMD1"

I am not sure why the "-05" is included in the actual PDF, but I have never seen articles or citations include the "-05" before. Also, in the header of each page in the PDF they don't use the "-05" but they do use "+AMD1:2015". I figured the ":2015" was redundant, what do you think?

Reply:
Will all our target file systems allow the '+' in the file name?

Reply:
Good point! I think "+" is allowed in all of them, but based on this SO post:

https://stackoverflow.com/questions/4814040/allowed-characters-in-filename

It seems we should stick with an underscore or a dash

Reply:
I will standardize on alphabet number, and underscore as it is harder to remember filenames that use both underscore and hyphen.